src/main/java/seedu/address/logic/commands/AddClaimCommand.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLAIM_AMOUNT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLAIM_ID;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INSURANCE_ID;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a claim to an existing person with existing Insurance Plan in the address book.
+ */
+public class AddClaimCommand extends Command {
+    public static final String COMMAND_WORD = "addClaim";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds an open claim to the insurance plan of the client identified "
+            + "by their client id. \n"
+            + "Parameters: INDEX (must be a positive integer), "
+            + " INSURANCE_PLAN_ID (must be a valid ID), "
+            + " Claim_ID (must be a string), "
+            + " Claim amount (must be a valid integer, remove the decimal point) \n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_INSURANCE_ID + " 0 "
+            + PREFIX_CLAIM_ID + " B1234 "
+            + PREFIX_CLAIM_AMOUNT + " 10000";
+
+    public static final String MESSAGE_SUCCESS =
+            "New claim added to Client: %1$s, under Insurance plan %2$s, with claimId: %3$s, claimAmount: %4$d";
```

Comment:
Should claimAmount be Claim Amount instead? To the user of the app we should not need to use camelCase

Reply:
yeah good spot!